R/combined.R

```diff
@@ -350,7 +350,7 @@ spim_population <- function(combined, ignore_uk = FALSE, by_age = TRUE,
 ##' @export
 spim_prop_infected <- function(combined, population,
                               regions = sircovid::regions("england")) {
-  idx_cum_infections <- combined$info[[1]]$info$index$cum_infections
+  idx_cum_infections <- combined$info[[1]]$info[[length(combined$info[[1]]$info)]]$index$cum_infections
```

Comment:
Sorry, this is awful, but probably the right thing to do at the moment. Probably this needs to be looked at more generally later (e.g., only saving the last epoch's data into the fit?)